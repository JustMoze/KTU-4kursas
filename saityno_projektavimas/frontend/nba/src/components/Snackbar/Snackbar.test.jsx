import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import 'jest-styled-components'
import AppSnackbar from './Snackbar'

describe('Snackbar', () => {
  const handleClose = jest.fn()
  const initProps = {
    message: 'Hello from test',
    open: true,
    severity: 'error',
    handleClose: handleClose,
  }
  const wrapper = shallow(<AppSnackbar {...initProps} />)

  it('AppSnackbar renders correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
