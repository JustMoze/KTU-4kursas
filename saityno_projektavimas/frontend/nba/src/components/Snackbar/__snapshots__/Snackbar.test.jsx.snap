// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snackbar AppSnackbar renders correctly 1`] = `
<WithStyles(ForwardRef(Snackbar))
  anchorOrigin={
    Object {
      "horizontal": "left",
      "vertical": "top",
    }
  }
  autoHideDuration={3000}
  onClose={[MockFunction]}
  open={true}
>
  <Alert
    onClose={[MockFunction]}
    severity="error"
  >
    Hello from test
  </Alert>
</WithStyles(ForwardRef(Snackbar))>
`;
